typings/index.d.ts

```diff
@@ -2786,6 +2786,8 @@ declare module 'discord.js' {
     rulesChannel?: ChannelResolvable;
     publicUpdatesChannel?: ChannelResolvable;
     preferredLocale?: string;
+    description?: string;
```

Comment:
Can't you also set descriptions to null?

Reply:
it does! fixed